### DISCUSSION TITLE
How to add support for iOS 9?

### DISCUSSION THREAD

**Discussion by rvalitov** [taiga-family/taiga-ui#848] [Q&A] [answered]
Hello! It turned out that my Angular 12 app now needs support for old iPad with iOS 9 on board. I enabled iOS 9 in `.browserslistrc` and enabled `web-animations-js` polyfill, but I still see a blank white page in Safari. Are there any specific Taiga related polyfills that will make it possible to run the app in iOS 9?

**Comment by waterplea** [comment, marked as answer]
We only support iOS 10 or higher, I'm sorry. iOS is terrible as it is and supporting Safari over 6 years old is plain impossible. We are even thinking of bumping it up in the next major release (hopefully). Is there a valid reason for this? All I can suggest is get a hold of such device and try remote debugging (if it was even there at that version) to see what exceptions are thrown. Or global catch with some alert to see what is the issue. But this would be very unstable. You can try polyfilling `IntersectionObserver` and `ResizeObserver` but I don't think it fails because of them.

**Reply by rvalitov** [reply]
The reason is in Apple device support policy. Even some ancient iPhone 5 runs iOS12. While much more recent iPads stay at iOS 9 and do not upgrade...
